mw-webapp/src/service/WayService.ts

```diff
@@ -55,9 +55,8 @@ export class WayService {
   /**
    * Update WayDTO
    */
-  public static async updateWayDTO(wayDTO: WayDTO, dayReportUuids: string[]) {
-    await updateDoc(doc(db, PATH_TO_WAYS_COLLECTION, wayDTO.uuid), {...wayDTO, dayReportUuids});
-
+  public static async updateWayDTO(wayDTO: WayDTO, uuid: string) {
+    await updateDoc(doc(db, PATH_TO_WAYS_COLLECTION, uuid), {...wayDTO});
```

Comment:
what happened here?

Reply:
I refactored request for update Way if change any property of model not only dayreports